examples/Getting Started/Export Image.ipynb

```diff
@@ -0,0 +1,223 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "%matplotlib inline\n",
+    "\n",
+    "import ee\n",
+    "ee.Initialize()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Exporting images, map tiles, features, and videos\n",
+    "\n",
+    "Shows how to export assets using Google Earth Engine API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [],
+   "source": [
+    "# prepare export parameters\n",
+    "dem = ee.Image(\"JAXA/ALOS/AW3D30_V1_1\")\n",
+    "region = ee.Geometry.LineString([[93.11668395996094, 28.709710285906034], [93.1486988067627, 28.673946616818256]]).bounds()"
```

Comment:
This would be easier to read of broken across multiple lines. This many significant digits is overkill.